src/Widgets/RoundedActor.vala

```diff
@@ -23,8 +23,13 @@ namespace Gala {
         }
 
         protected virtual bool drawit (Cairo.Context ctx) {
+            var buffer_width = (int)this.width;
+            var buffer_height = (int)this.height;
+            if (buffer_width == 0 || buffer_height == 0)
+                return false;
```

Comment:
More of a question than an issue. Why returning false here? The docs say:

> true if the signal emission should stop, and false otherwise

Does the size eventually change and it gets repainted?